ext/drcovlib/modules.c

```diff
@@ -466,6 +461,11 @@ module_table_entry_print(module_entry_t *entry, char *buf, size_t size)
 #endif
     read_entry.path = full_path;
     read_entry.custom = entry->custom;
+#ifndef WINDOWS
+    // For unices we record the physical offset from the backing file. Each
+    // module entry only has a sigle segment so we grab index 0 of the array.
+    read_entry.offset = entry->data->segments[0].offset;
```

Comment:
I don't think this is correct: entry->data points at DR's module_data_t and each segment entry here points at the same data structure.  `module_entry_t` needs to store either the offset or the segment index (or I guess you could compute the segment index here by assuming they're contiguous and in order).

Reply:
Done.